### ISSUE TITLE
Confused on how to pass arguments to query

### ISSUE THREAD

**Issue by Superpat** [apollographql/apollo-client#1980]
So I'm pretty sure I'm simply confused as to how this all works. But I've tried many combinations and I cant get it to work.

**Intended outcome:**

Trying to query a specific object with an id
```
function ShowItem({ownProps, data: { loading, error, item }}) {
    return <p>item.name</p>;
}

const item = gql`
    query item($num: Int!) {
        item(num: $num) {
            num
            name
            price
        }
    }`;

export default graphql(item, {
    options: ({ num }) => ({
        variables: {
            num: num
        }
    }),
    props: ({ownProps, data}) => ({
        ownProps,
        data
    })
})(ShowItem);
```

**Actual outcome:**

When used like this:
```
<ShowItem num={num} />
```
I get a 400 error becauses arguments to the graphql query are not being passed along.

**Version**
- apollo-client@<1.8.1>

**Comment by Fxlr8** [comment]
It is better to ask such questions on stackoverflow.com
You should get num from props
Try this
```
options:
    props => ({
        variables: {
            num: props.num
        }
    })
```

**Comment by Superpat** [comment]
Wow I just realized that I was modifying viewItem.js instead of ViewFile.js.... Spent so much time on this I feel like the biggest fool!

Oh well, at least your response lead me to find out that there couldnt possibly be something wrong with the code (As I had tried the way you wrote it before)

I've always preferred using github issues directly compared to stackoverflow, as I find the community on stack overflow to be slightly hostile at times. But I will abstain from posing this kind of question here in the future. Thanks for the help!
